Client/Phpredis/Client.php

```diff
@@ -13,7 +13,7 @@
 
 namespace Snc\RedisBundle\Client\Phpredis;
 
-use Redis;
+use Predis\Client as Redis;
```

Comment:
Error reported before hotfix:
Fatal error: Class 'Redis' not found in /vendor/snc/redis-bundle/Client/Phpredis/Client.php on line 26

Stack: 
"snc/redis-bundle": "^2.1",
"predis/predis": "^1.1",

used in context with monolog and elastica to save logs on elasticsearch.

Work after hotfix.

Reply:
I think the bundle was configured to use phpredis instead of predis no?
Can you show the config please?
Can you create a reproducer if you are sure? So that we can all speak about the same thing.

Reply:
This scenario with monolog send log messages to elasticsearch. Monolog configuration example:

monolog:
  channels: ['main']
  handlers:
    main:
      type: elasticsearch
      level: debug
      tags: monolog.logger
      id: Monolog\Handler\ElasticSearchHandler
      formatter: monolog.formatter.elastica
      channels: ["!event"]
      index: monolog
      document_type: logs
      ignore_error: true
      elasticsearch:
        host: elk
        port: 9200

Reply:
How is this related to redis?
Can you please provide your snc redis bundle config?

Reply:
Please format your snippets next time.

Reply:
I cannot reproduce the issue. Could you set up a repository that would reproduce it?